src/content/api/loyalty-programs.mdx

```diff
@@ -79,6 +80,16 @@ A loyalty program is a container for one or more [Promotions](/api/promotions),
       The id of the [Media Upload](/api/media-uploads/) image of the Loyalty Program.
     </Property>
   </Properties>
+
+  <Properties heading="Errors">
+    <Error code="403" message="INVALID_ACCOUNT_TYPE">
+      The Account is of the wrong type.
+    </Error>
+
+    <Error code="403" message="ACCOUNT_NOT_FOUND">
```

Comment:
We don't return ACCOUNT_NOT_FOUND messages because it provides a way for baddies to try and guess accountIds